package/boot/uboot-rockchip/patches/109-rockchip-add-FriendlyElec-NanoPi-R3S.patch

```diff
@@ -0,0 +1,758 @@
+--- /dev/null	2024-10-08 20:38:15.835000058 -0400
```

Comment:
please refresh uboot patch with `make package/uboot-rockchip/{clean,refresh} V=s`
maybe this patch should start with `100-`.